### PR TITLE
Bug 9235 missing pdf form text

### PR DESCRIPTION
**Pull request by shumway-tm**
Fixes bug to capture PDF Form Fields.

### CODE REVIEW

**Review by TomElliottFlexion — APPROVED** [review]

**Review by mmarcotte — APPROVED** [review]

**Review by ujahio — APPROVED** [review]

**Review by jimlerza — APPROVED** [review]

File: shared/src/business/useCases/pdf/sanitizePdfInteractor.test.js [hunk 1]
```diff
@@ -45,6 +52,21 @@ describe('sanitizePdfInteractor', () => {
     ).toBe(mockKey);
   });
 
+  it('calls setPdfFields when PDF has form fields', async () => {
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/pdf/sanitizePdfInteractor.test.js]: I really like this test name! Does this mean that the call to `applicationContext.getUseCaseHelpers().setPdfFields(fields)` should be moved into the if check on line 42?

File: shared/src/business/useCases/pdf/sanitizePdfInteractor.js [hunk 2]
```diff
@@ -1,10 +1,27 @@
+/**
+ * setPdfFields helper function
+ *
+ * @param {array} fields array of pdf form fields
+ */
+
+exports.setPdfFields = fields => {
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/pdf/sanitizePdfInteractor.js]: I love that this was added as a use case helper! 

Nit: Could it be extracted into its own file in the `useCaseHelpers` folder?

File: shared/src/business/useCases/pdf/sanitizePdfInteractor.test.js (hunk 1 above)
Comment (TomElliottFlexion) [reply shared/src/business/useCases/pdf/sanitizePdfInteractor.test.js]: Good call!

File: shared/src/business/useCases/pdf/sanitizePdfInteractor.test.js [hunk 3]
```diff
@@ -26,6 +30,9 @@ describe('sanitizePdfInteractor', () => {
     });
 
     applicationContext.getPersistenceGateway().deleteDocumentFromS3 = jest.fn();
+
+    applicationContext.getUseCaseHelpers().setPdfFields = jest.fn();
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/pdf/sanitizePdfInteractor.test.js]: Nit: `setPdfFields` and `deleteDocumentFromS3` could be added to the `createTestApplicationContext` to follow our "mocking strategy" a little more closely

File: shared/src/business/useCases/pdf/sanitizePdfInteractor.js (hunk 2 above)
Comment (TomElliottFlexion) [reply shared/src/business/useCases/pdf/sanitizePdfInteractor.js]: Let's do it

**Review by rachelschneiderman — APPROVED** [review]

File: shared/src/business/useCases/pdf/sanitizePdfInteractor.test.js (hunk 3 above)
Comment (shumway-tm) [reply shared/src/business/useCases/pdf/sanitizePdfInteractor.test.js]: Thanks for the heads up! Fixed it!

**Pull request merged**